learn/teams/manage-roles.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Manage Roles
+id: ""
+---
+---
+
+It helps define custom roles (e.g. resource managers/developers) to give fine-grained access privileges to projects. And assign user roles like resource managers/developers to the members of your team's projects. Customize the access permissions for each role as you want. A user can perform actions on a project based on their assigned role & permissions.
```

Comment:
Give some breif about how does it  different from roles of users from User section.